main.go

```diff
@@ -40,6 +41,16 @@ func doc() {
 	for _, name := range names {
 		resource := resources[name]
 		fmt.Printf("\n### %s\n\n", name)
+		if strings.HasSuffix(name, "_grant") {
+			grant_resource_name := strings.Trim(name, "_grant")
+			fmt.Printf(`
+			**Note**: The grant resource create exclusive attachments of grants.
+			Across the entire AWS account, all of the users/roles/groups to which a single grant is attached must be declared
```

Comment:
`users/roles/groups` might not make sense in this context -> I wonder if you can fetch it from the schema that is available in the context